src/main/java/team/chisel/block/BlockHempcreteSand.java

```diff
@@ -0,0 +1,18 @@
+package team.chisel.block;
+
+import net.minecraft.block.Block;
+
+import com.cricketcraft.chisel.api.ChiselTabs;
+
+public class BlockHempcreteSand extends BlockCarvableSand {
+
+    public BlockHempcreteSand() {
+        super();
+        setCreativeTab(ChiselTabs.tabStoneChiselBlocks);
+        setHardness(0.5F);
+        setResistance(0.5F);
+        setStepSound(Block.soundTypeSand);
```

Comment:
No need to create a class for this block, just call these method after instantiation of `BlockCarvableSand`, similar to Cubit.